src/api/routes/termsAndConditions.js

```diff
@@ -0,0 +1,15 @@
+import { Router } from 'express';
+import checkValidToken from '../../middlewares/checkValidToken';
+import termsAndConditions from '../controllers/termsAndConditionsController';
+import errorHandler from '../../middlewares/errorHandler';
+import isAdmin from '../../middlewares/IsAdmin';
+
+const termsAndConditionsRouter = new Router();
+
+termsAndConditionsRouter.get('/:id', errorHandler(termsAndConditions.getTermsAndConditions));
```

Comment:
@murediane 
1. why do you need an `id` field here? 
2. Do you need to be an admin to view terms and conditions?

Reply:
because I did not want to hard code the id in case the table has more versions of terms and conditions .i was getting by terms and conditions Id. 
Do you think I should change that

Reply:
Admin can view or update .the rest can only view

Reply:
Alright, your point is valid.